app/src/main/java/org/oppia/android/app/story/StoryFragmentPresenter.kt

```diff
@@ -142,6 +150,30 @@ class StoryFragmentPresenter @Inject constructor(
             ).parseOppiaHtml(
               storyItemViewModel.summary, binding.chapterSummary
             )
+          if (storyItemViewModel.chapterSummary.chapterPlayState
+            == ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES
+          ) {
+            val missingPrerequisiteSummary = fragment.getString(
+              R.string.chapter_prerequisite_title_label,
+              storyItemViewModel.index.toString(),
+              storyItemViewModel.missingPrerequisiteChapter.name
+            )
+            val ss = SpannableString(missingPrerequisiteSummary)
```

Comment:
Rename ss to `chapterLockedSpannable` or something which can explain the context.

Reply:
Done.